cypress/support/fragments/circulation/newNoticePolicyTemplate.js

```diff
@@ -40,7 +40,9 @@ export default {
   },
 
   create: (noticePolicyTemplate, autoSave = true) => {
+    cy.wait(4000);
     cy.get('#input-patron-notice-name').type(noticePolicyTemplate.name);
+    cy.wait(5000);
```

Comment:
Why need here to wait?

Reply:
this waiter is needed to synchronize with the page.  we need to wait until the field #input-patron-notice-name is shown and ready to receive the text. Without of this waiter only the part of the entered string was saved.